From 16c10c8949b8c520a721bc5e5527f8bd0cf9a3c7 Mon Sep 17 00:00:00 2001
From: Han Xu <b45815@freescale.com>
Date: Tue, 9 Jun 2015 10:27:00 -0500
Subject: [PATCH 0624/5242] MLK-11074: ARM: dts: fix mipi pin conflict with
 ecspi1

commit  99f9724af9c886bdae05afa6c1baf64b0d6333a8 from
https://source.codeaurora.org/external/imx/linux-imx.git

mipi pins conflict with ecspi1 on i.MX7D 12x12 lpddr3 ARM2 board, manage
the mipi pins in a individual group to solve the issue.

Signed-off-by: Han Xu <b45815@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index 746d8d6..c07a113 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -565,7 +565,11 @@
 				MX7D_PAD_SD2_WP__GPIO5_IO10       0x59
 				MX7D_PAD_SD2_RESET_B__GPIO5_IO11  0x59
 				MX7D_PAD_GPIO1_IO13__GPIO1_IO13   0x59
+			>;
+		};
 
+		pinctrl_hog_mipi: hoggrp_mipi {
+			fsl,pins = <
                                 MX7D_PAD_ECSPI1_SCLK__GPIO4_IO16  0x59
                                 MX7D_PAD_ECSPI1_MOSI__GPIO4_IO17  0x59
 			>;
-- 
1.7.9.5

